css/frontPage.css

```diff
@@ -143,3 +143,24 @@ footer {
     clear:             right;
     box-shadow:        4px 4px 2px rgba(0, 0, 0, 0.5);
 }
+
```

Comment:
Could you remove the two extra blank lines and align the CSS property values in a column like the above CSS?